src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummmary", () => {
+  const validProps = {
+    date: 1111111111111,
+    description: "Stub description",
+    icon: "800",
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+    onSelect: () => {},
+  };
+
+  const formattedDate = new Date(validProps.date).toDateString().slice(0, 10);
+
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <ForecastSummary
+        props={validProps}
+        date={validProps.date}
+        description={validProps.description}
+        icon={validProps.icon}
+        temperature={validProps.temperature}
+        onSelect={validProps.onSelect}
+      />
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders correct values for props", () => {
+    const { getByText, getByTestId } = render(
+      <ForecastSummary
+        date={validProps.date}
+        description={validProps.description}
+        icon={validProps.icon}
+        temperature={validProps.temperature}
+        onSelect={validProps.onSelect}
+      />
+    );
+
+    expect(getByText(formattedDate)).toHaveAttribute(
```

Comment:
With react testing, we're really more concerned with what the user sees on the page rather the hidden things that we as devs care more about (such as classes). Instead of looking for a class attribute like this could we not check if the text just exists in the document instead?